flickit-assessment-core/src/main/java/org/flickit/assessment/core/application/domain/AttributeValue.java

```diff
@@ -168,4 +168,17 @@ private double calcConfidenceGainedScore(Map<Long, Double> questionIdToWeightMap
     public Double getWeightedConfidenceValue() {
         return confidenceValue * attribute.getWeight();
     }
+
+    public Map<Long, Double> getWeightedScore() {
```

Comment:
Move this to after the `getWeightedLevel()` method.
The methods in the second half of the file are related to calculating confidence.